Mathlib/Topology/UniformSpace/Cauchy.lean

```diff
@@ -20,7 +20,7 @@ open Filter TopologicalSpace Set Classical UniformSpace Function
 
 open Classical Uniformity Topology Filter
 
-variable {α : Type u} {β : Type v} [UniformSpace α]
+variable {α : Type u} {β : Type v} [uniformSpace : UniformSpace α]
```

Comment:
Why did you add this name?

Reply:
To allow for `Cauchy (uniformSpace := u)` instead of `@Cauchy _ u`, as well as the `(uniformSpace := _)` dark magic (instead of `@`s and `_`s). I'm not sure the benefit is very clear, but I *think* it is more readable.

Reply:
Oh I see. Maybe we should have a general strategy about this, but in this case it clearly helps since I was able to understand the proofs without realizing this trick was needed..